src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  if(days<3){
+    return days*RENT_COST;
+  } else if(days < 7){
+    return days*RENT_COST-20;
+  }
+  return days*RENT_COST-40;
```

Comment:
The discount value `40` should be a named constant, and it seems to be incorrect. According to the task description, the discount for 7 or more days should be `$50` off, not `$40`. Please correct the discount value and use a named constant.